### ISSUE TITLE
Error: invalid option: --token=xxx

### ISSUE THREAD

**Issue by KevinBongart** [buildkite/homebrew-buildkite#14]
Hi,

I followed the setup instructions when signing up for Buildkite but got this error:

```sh
$ brew tap buildkite/buildkite
$ brew install --token='xxx' buildkite-agent

Usage: brew install [options] formula

Install formula. Additional options specific to formula may be appended to
the command.

[...]

Error: invalid option: --token=xxx
```

I also tried with appending the option to the command:

```sh
$ brew install buildkite-agent --token='xxx'
```

but got the same error.

I ended up configuring the `buildkite-agent.cfg` file, which worked, but thought you should know.

**Comment by ticky** [comment]
Hi @KevinBongart, are you able to give us the output from these commands:

To get the version of Homebrew:
`brew --version`

To get the version of the buildkite tap:
`(cd $(brew --repo buildkite/buildkite); git show)`

To get info about the buildkite-agent formula:
`brew info buildkite-agent`

The output of `brew tap buildkite/buildkite` would be useful too!

**Comment by KevinBongart** [comment]
Hi @ticky, I just tried again and it worked. Must have been an outdated brew version (though I usually try to keep it up to date):

```sh
$ brew uninstall buildkite-agent
Uninstalling /usr/local/Cellar/buildkite-agent/3.21.0... (5 files, 31.2MB)

Warning: The following buildkite-agent configuration files have not been removed!
If desired, remove them manually with `rm -rf`:
  /usr/local/etc/buildkite-agent
  /usr/local/etc/buildkite-agent/bootstrap.sh
  /usr/local/etc/buildkite-agent/buildkite-agent.cfg
  /usr/local/etc/buildkite-agent/hooks
  /usr/local/etc/buildkite-agent/hooks/checkout.sample
  /usr/local/etc/buildkite-agent/hooks/command.sample
  /usr/local/etc/buildkite-agent/hooks/environment.sample
  /usr/local/etc/buildkite-agent/hooks/post-artifact.sample
  /usr/local/etc/buildkite-agent/hooks/post-checkout.sample
  /usr/local/etc/buildkite-agent/hooks/post-command.sample
  /usr/local/etc/buildkite-agent/hooks/pre-artifact.sample
  /usr/local/etc/buildkite-agent/hooks/pre-checkout.sample
  /usr/local/etc/buildkite-agent/hooks/pre-command.sample
  /usr/local/etc/buildkite-agent/hooks/pre-exit.sample

$ rm -rf  /usr/local/etc/buildkite-agent /usr/local/etc/buildkite-agent/bootstrap.sh /usr/local/etc/buildkite-agent/buildkite-agent.cfg /usr/local/etc/buildkite-agent/hooks /usr/local/etc/buildkite-agent/hooks/checkout.sample /usr/local/etc/buildkite-agent/hooks/command.sample /usr/local/etc/buildkite-agent/hooks/environment.sample /usr/local/etc/buildkite-agent/hooks/post-artifact.sample /usr/local/etc/buildkite-agent/hooks/post-checkout.sample /usr/local/etc/buildkite-agent/hooks/post-command.sample /usr/local/etc/buildkite-agent/hooks/pre-artifact.sample /usr/local/etc/buildkite-agent/hooks/pre-checkout.sample /usr/local/etc/buildkite-agent/hooks/pre-command.sample /usr/local/etc/buildkite-agent/hooks/pre-exit.sample

$ brew install --token='xxx' buildkite-agent
==> Installing buildkite-agent from buildkite/buildkite
==> Downloading https://github.com/buildkite/agent/releases/download/v3.22.0/buildkite-agent-darwin-amd64-3.22.0.tar.gz
==> Downloading from https://github-production-release-asset-2e65be.s3.amazonaws.com/16347164/e8a2ae80-96a4-11ea-994d-27b14df6fe48?X-Amz-Algo
######################################################################## 100.0%
==> Caveats
buildkite-agent is now installed!
```

Also:

```sh
$ brew --version
Homebrew 2.3.0-53-g13caf6e
Homebrew/homebrew-core (git revision 467e0; last commit 2020-06-05)
Homebrew/homebrew-cask (git revision 8a0acb; last commit 2020-06-05)

$ (cd $(brew --repo buildkite/buildkite); git show)
commit da9dbb4514a8ec1d73a10c6fd2c70ed37083316f

$ brew info buildkite-agent
buildkite/buildkite/buildkite-agent: stable 3.22.0
```

Sounds like this was related to Homebrew 2.3.0 and fixed with #16 :)
